src/inverseRobot.js

```diff
@@ -8,6 +8,27 @@
 
 function inverseRobot(robot) {
   // write code here
+  let tempKyesRobot = {};
+  let tempValueRobot = {};
+
+  tempKyesRobot = Object.keys(robot);
```

Comment:
There is no need to declare 'tempKyesRobot' as an empty object and then assign it with 'Object.keys(robot)'. You can directly assign it like this: 'let tempKeysRobot = Object.keys(robot);'.